Gopkg.toml

```diff
@@ -42,6 +42,10 @@ required = [
   name = "google.golang.org/grpc"
   version = "^1.8.2" # match istio/api
 
+[[constraint]]
+  name = "github.com/envoyproxy/go-control-plane"
+  revision = "cd217031c55a80342a864f9aff3b7a7f22205891"
+
```

Comment:
something is off.. I already added this. So why is this appearing as new for you?

Reply:
merge issue. I've now taken care of it.